src/component/elements/Input.tsx

```diff
@@ -0,0 +1,198 @@
+import debounce from 'lodash/debounce';
+import { useState, useEffect, useCallback, useRef, CSSProperties } from 'react';
+
+import { forwardRefWithAs } from '../../utils';
+
+const styles: Record<'label' | 'input', CSSProperties> = {
+  label: {
+    lineHeight: 2,
+    userSelect: 'none',
+  },
+  input: {
+    height: '100%',
+    width: '100px',
+    borderRadius: '5px',
+    borderWidth: '0.55px',
+    borderColor: '#c7c7c7',
+    borderStyle: 'solid',
+    margin: '0px 5px 0px 5px',
+    textAlign: 'center',
+  },
+};
+
+export interface InputProps
+  extends React.InputHTMLAttributes<HTMLInputElement> {
+  name?: string;
+  label?: string;
+  value?: any;
+  styleInput?: {
```

Comment:
You forgot to change the prop name in `ManualPhaseCorrectionPanel.jsx`, but I'm not sure why you renamed the prop to `styleInput`? Is it because of the collision with `InputHTMLAttributes` ? In that case you can do `InputProps extends Omit<React.InputHTMLAttributes<HTMLInputElement>, 'style'>`